eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -332,9 +332,15 @@ jobs:
 
       # Generate Documentation
       - pwsh: |
-          $setupFile="doxygen-1.8.15-setup.exe";
-          Invoke-WebRequest -MaximumRetryCount 10 -Uri "https://azuresdktooling.blob.core.windows.net/dependencies/doxygen-1.8.15-setup.exe" `
-          -OutFile $setupFile | Wait-Process; Start-Process -Filepath .\$setupFile -ArgumentList @("/VERYSILENT") -Wait
+          $setupFile="doxygen-1.8.18-setup.exe";
+
+          Invoke-WebRequest `
+            -MaximumRetryCount 10 `
+            -Uri "https://azuresdktooling.blob.core.windows.net/dependencies/$setupFile" `
+            -OutFile $setupFile `
+          | Wait-Process;
+
+          Start-Process -Filepath .\$setupFile -ArgumentList @("/VERYSILENT") -Wait
```

Comment:
I am a bit confused by this change vs the title of this PR. How does this PR, `fix missing README.md headers`?

Is the doxygen version update what fixes it? What was the issue?

Reply:
Yes, it was a doxygen bug fixed in their latest bits